package.json

```diff
@@ -206,7 +206,7 @@
     "generate-changelog": "git log ...$(git tag | tail -n 1) --pretty=format:'- %s' --reverse"
   },
   "dependencies": {
-    "@eagleoutice/flowr": "^2.0.5"
+    "@eagleoutice/flowr": "^2.0.9"
```

Comment:
we should denote such dep updates somewhere in the changelog / have specifc dep prs just for this so that it is easier to track the related flowr version.

Reply:
True! There's still the annoying changelog detection issue (#82), so until then, the changelog is kinda useless tbh :(